static/js/components/tiles/donut_tile.tsx

```diff
@@ -114,9 +114,18 @@ export function DonutTile(props: DonutTilePropType): JSX.Element {
       replacementStrings={getReplacementStrings(props, donutChartData)}
       className={`${props.className} bar-chart`}
       allowEmbed={true}
-      getDataCsv={
-        donutChartData ? () => dataGroupsToCsv(donutChartData.dataGroup) : null
-      }
+      getDataCsv={() => {
+        // Assume all variables will have the same date
+        const date =
```

Comment:
may be a good idea to extract out a `getDate` function that gets used by both the regular data fetch and the get csv call to make sure that the logic for getting date is the same in both cases. Because if you look at the fetchData call in this file, it looks like it never actually fetches data for a specific date.

(Here and in all the other tiles)

Reply:
Updated and added a todo to specify dates independently for each stat var